editor/src/core/workers/parser-printer/parser-printer.ts

```diff
@@ -654,6 +655,74 @@ export const printCode = memoize(printCodeImpl, {
   maxSize: 1,
 })
 
+function isVariableDeclaration(line: string): boolean {
```

Comment:
This looks really brittle to me! It's probably fine for this use case but I can certainly imagine this is going to break on slightly different code.

Reply:
Yeah, I think this whole PR is one of those "it's not great but better than nothing" situations unfortunately :grimacing:

Reply:
That's cool.